### DISCUSSION TITLE
Bootstrap .js errors in Eclipse IDE

### DISCUSSION THREAD

**Discussion by brunogenovese** [twbs/bootstrap#36102] [Q&A]
I downloaded the latest Bootstrap and installed the files in my Eclipse 2021-12 project css and js folders.  

Immediately Eclipse complained about errors in bootstrap.bundle.js, bootstrap.ems.js and bootstrap.js.  

My guess is that either Eclipse is defaulting to a different version of Javascript, or that Bootstrap is built to the looser Microsoft standards of coding and it is triggering Eclipse's error detection.

Does anybody know how to adjust Eclipse to make the errors in Bootstrap's js go away?

**Comment by bestOf78** [comment]
Hi Bruno, did you find a solution to your problem ? I have the same one.
Thank you in advance.

**Reply by brunogenovese** [reply]
I think solve it although I can't remember how.  But TBH after about a decade or two of using Eclipse I finally gave up on frustration due to its many maven and gradle integration issues and finally switched to IntelliJ.

**Reply by bestOf78** [reply]
Thank you anyway and good lock with IntelliJ.

**Reply by PrabalGhosh07** [reply]
Hii bestof78 ,did you find a solution to your problem ? I have the same one.

**Comment by WinterSilence** [comment]
[Specifying the JVM](https://wiki.eclipse.org/Eclipse.ini)

**Reply by bestOf78** [reply]
Thank you for your request. Specifying the JVM path in eclipse.ini file didn't solve my problem. I still have errors in all js files of bootstrap.
![image](https://user-images.githubusercontent.com/6182040/195292902-ecfb84b8-1a5b-4bb5-9d2e-44e4cdecc4e8.png)

**Reply by bestOf78** [reply]
*response

**Reply by crlabrada10** [reply]
Hi bestOf78, did you find a solution to your problem ? I have the same one.
Thank you in advance in eclipse

**Reply by om-learn** [reply]
Having the same Problem, did anyone ever found a solution?

**Comment by junjun-dachi** [comment]
Right Click on "Project" -> "Properties" -> "Validation" -> "JavaScript Validation" / "Settings" -> "Add Exclude Group" -> Click on the Exclude Group and "Add Rule" -> choose "Folder or file name" -> select Bootstrap folder.

**Reply by Asunro** [reply]
Juste created an account for this : I had the same problem and this workaround worked perfectly - thanks very much !
